src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import fetchGithubUser from './utils/githubFetch';
+import UserList from './Components/UserList';
+import './App.css';
+
+const mapToUsersArr = (str) => {
+  return str.replace(/\s+/g, '').split(',');
+};
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      userInput: '',
+      usersTofetch: [],
+      showUserList: false,
+      githubUsers: []
+    }
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
+  }
+
+  updateUserList(users) {
```

Comment:
Right now, it always queries the `github` api for users. One way to make it more performant would be to use memoization. I would basically cache the result for users that have been queried before on the user's device (most likely using `localStorage`) and return the cached result if the user requests for it again.